.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: minimal
+cache: bundler
```

Comment:
Is this going to do anything? All the installation's done inside the container.

Reply:
Thought it was caching gems in the local directory but can't see any evidence of why I thought that now, I'll remove.